.github/workflows/build-and-push-async-upload.yml

```
@@ -59,3 +59,21 @@ jobs:
         tags: ${{ env.IMG_REGISTRY }}/${{ env.IMG_ORG }}/${{ env.IMG_NAME }}:${{ env.VERSION }}
         cache-from: type=gha
         cache-to: type=gha,mode=max
+
+    - name: Tag Latest
+      # this workflow works on main branch and tags, so we need to tag as 'latest' and 'main'
+      shell: bash
+      env:
+        IMG: "${{ env.IMG_REGISTRY }}/${{ env.IMG_ORG }}/${{ env.IMG_NAME }}"
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)